### ISSUE TITLE
PDF report from command line

### ISSUE THREAD

**Issue by rolivella** [vdemichev/DiaNN#656]
Hi, is it possible to generate the PDF report while running DIANN from command line? I'm using this version: 

```
DIA-NN 1.8 (Data-Independent Acquisition by Neural Networks)
Compiled on Jun 28 2021 10:59:57
```

Thanks!

**Comment by vdemichev** [comment]
Yes, just need to replicate the command the GUI prints at the end of the log.

**Comment by rolivella** [comment]
You mean this commnad? 

`DIA-NN-plotter.exe "H:\tmp\report.stats.tsv" "H:\tmp\report.tsv" "H:\tmp\report.pdf"`

**Comment by kostrouc** [comment]
What is the equivalent in the linux version? 
/usr/diann/1.8.1/ folder only contains diann-1.8.1 executable. I don't see a DIA-NN-plotter executable or a parameter in the "command-line reference" guide.
